src/styles.scss

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
```

Comment:
N o need for components as of this moment

Reply:
done sir